### DISCUSSION TITLE
Mac 系统安装 vitables时候出现问题

### DISCUSSION THREAD

**Discussion by MeAPikachu** [physics-data/faq#42] [Q&A] [answered]
Mac 使用 pip3 install vitables 命令， 无法安装（其他软件可以顺利安装）
<img width="750" alt="Screenshot 2021-08-18 at 11 33 07" src="https://user-images.githubusercontent.com/61261422/129832719-27b73fad-4257-4e39-baa5-a6dee656248a.png">
报错内容过长,难以截图
<img width="476" alt="Screenshot 2021-08-18 at 11 34 28" src="https://user-images.githubusercontent.com/61261422/129832868-a5c2813a-5b24-422d-a9d0-0239371cbbfc.png">

**Comment by jiegec** [comment, marked as answer]
ViTables 不支持 macOS

**Reply by jiegec** [reply]
可以试试 Panoply：https://www.giss.nasa.gov/tools/panoply/download/
